plenum/common/util.py

```diff
@@ -335,7 +335,10 @@ def friendlyToRaw(f):
 
 
 def cryptonymToHex(cryptonym: str) -> bytes:
```

Comment:
Its a low level library function, it should be throwing errors, the handling should move to the code using this